### ISSUE TITLE
Modpack Permission: Modrinth

### ISSUE THREAD

**Issue by Sukasa** [pWn3d1337/Techguns#282]
In your [modpack license document](https://github.com/pWn3d1337/Techguns/wiki/Modpack-Policy), you list a number of modpack hosts as automatically allowed.  Is Modrinth also an acceptable platform for which you will allow TechGuns (for 1.7.10) to be part of a modpack?

**Comment by pWn3d1337** [comment]
Yeah Modrinth is fine. License was written before Modrinth was a thing.
